src/components/ui/command-copier/command-copier.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import modifiers from '../../../utils/modifiers';
+import MetaDataValue from '../../metadata/metadata-value';
+import IconButton from '../../icon-button';
+import CopyIcon from '../../icons/copy';
+import './command-copier.css';
+
+const CommandCopier = ({ runCommand }) => {
+  const [showCopied, setShowCopied] = useState(false);
+
+  const onCopyClick = () => {
+    window.navigator.clipboard.writeText(runCommand);
+    setShowCopied(true);
+    setTimeout(() => setShowCopied(false), 1500);
+  };
+
+  return (
+    <div className="container">
+      <MetaDataValue
+        container={'code'}
+        className={modifiers('run-command-value', {
+          visible: !showCopied,
+        })}
+        value={runCommand}
+      />
+      {window.navigator.clipboard && runCommand && (
+        <>
+          <span
+            className={modifiers('copy-message', {
+              visible: showCopied,
+            })}
+          >
+            Copied to clipboard.
+          </span>
+          <ul className="toolbox">
+            <IconButton
+              ariaLabel="Copy run command to clipboard."
```

Comment:
Is this the caption when I hover over the copy button? I don't actually see it on my screen so not sure when it's meant to appear?

Either way, I think this label is specific to the old use of the copying `kedro run` and doesn't sound right if this is now a generic button also used for the `pip install` command. Maybe just "Copy command to clipboard" or even "Copy to clipboard" would be better.

The variable name `runCommand` should maybe be updated to just to `command` or maybe even more generic since we could potentially copy something else with this button too (like a dataset filepath)?

Reply:
the `aria-label` is mainly for accessibility 😄  though I agree we can change `run command` to just `command` to allow this to be more specific.